db-remove

```diff
@@ -45,6 +44,16 @@ for pkgbase in "${pkgbases[@]}"; do
 done
 
 for tarch in "${tarches[@]}"; do
-	arch_repo_modify remove "${repo}" "${tarch}" "${remove_pkgs[@]}"
+	if (( ${#remove_pkgs[@]} >= 1 )); then
+		arch_repo_modify remove "${repo}" "${tarch}" "${remove_pkgs[@]}"
+		for pkg in "${remove_pkgs[@]}"; do
+			if is_globfile "${FTP_BASE}/${repo}-debug/os/${tarch}/${pkg}-debug"*; then
```

Comment:
Same here, we'll need to track it when building `remove_pkgs`

Reply:
Wrote up a suggestion